menu.py

```diff
@@ -1009,3 +797,85 @@ def get_game_config(self) -> GameConfig:
         """
         return GameConfig(self.player_count, self.faction_colours[self.faction_idx][0], self.biome_clustering_enabled,
                           self.fog_of_war_enabled, self.climatic_effects_enabled)
+
+    def next_menu_option(self, current_option: typing.Type[Enum], wrap_around: bool = False) -> None:
```

Comment:
Getting some warnings about this Enum type hint - might be easier to change it from `typing.Type[Enum]` to `SetupOption | WikiOption | MainMenuOption | VictoryType`?

Reply:
[841bd66](https://github.com/ChrisNeedham24/microcosm/pull/91/commits/841bd66dbd0cc1c655d54ad409b59603d697ddd2)